docs/manage/config-policies.rst

```diff
@@ -0,0 +1,179 @@
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies using either the WebUI or the CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at either the cluster or workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level, follow these steps:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level, follow these steps:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Here are some example policies:
+
+#. Limit resources.
+
+In the following example for an Agent resource manager (RM), the ``priority_limit`` is set to ``15``
+and ``max_slots`` is set to ``1``. This means a user cannot set a priority value lower than 15 and
+cannot set ``max_slots`` to greater than ``1``.
+
+.. note::
+
+   Tasks (NTSC - notebooks, Tensorboards, shells, and commands) and some resource managers (RMs)
+   have a priority. Priority levels are between 1 and 99. The default cluster priority is usually
+   42. The priority limit determines the order in which queued tasks run. In this example we are
+   using limits because for the agent resource manager, a lower number means a higher priority. For
+   a Kubernetes resource manager, a higher number means a higher priority.
+
+.. code:: bash
+
+   # Set priority limit to 15
+   constraints:
+      priority_limit: 15
+      resources:
+         max_slots: 1
+
+If your priority limits are above the defaults, for example if your notebook priority is set to 10
+and the limit is 15, then the request fails and the system displays a descriptive message.
+
+#. Limit maximum GPU usage per experiment:
```

Comment:
I don't think any of these examples make sense; I don't think they have correct formatting or use valid fields. Are they just placeholders?

Reply:
@kkunapuli yes the doc was created using placeholder text, the code has not been reviewed, can you review it and help make the example make sense?